### PR TITLE
add retries to status check

### PR DESCRIPTION
**Pull request by rayashworth**

### CODE REVIEW

File: ansible/roles/request_ocp_fyre/tasks/request-ocp-fyre.yml [hunk 1]
```diff
@@ -66,6 +66,8 @@
     force_basic_auth: "{{ fyre_force_basic_auth }}"
   changed_when: false
   register: error_status
+  retries: 12
```
[the remark this answers is not in the document]
Comment (rayashworth) [reply ansible/roles/request_ocp_fyre/tasks/request-ocp-fyre.yml:69]: it has been

[the remark this answers is not in the document]
Comment (rayashworth) [reply ansible/roles/request_ocp_fyre/tasks/request-ocp-fyre.yml:70]: I added the until

**Review by gmarcy — APPROVED** [review]
LGTM

**Review by websterc87 — APPROVED** [review]
Looks good to me as well

**Pull request merged**